LandSandBoat/server
# [sql] Audit prices of arrows/quivers/toolbags/ninja tools/shurikens/shuriken pouches

**_I affirm:_**

## What does this pull request do?

Audits various sell prices (and adds some missing ones)

## Steps to test these changes

Check prices on some items, verify SQL imports properly

Sources:
![](https://cdn.discordapp.com/attachments/837796461877526533/999412442167779388/Ammunition_category_Curio_Moogle_-_Challoux.jpg)
![](https://cdn.discordapp.com/attachments/837796461877526533/999423536722616370/Nijutsu_category_Curio_Moogle_-_Challoux.jpg)
from Challoux provided by @Uyngh
